src/ray/util/logging.h

```diff
@@ -258,10 +258,23 @@ class RayLog {
   ///
   /// \parem appName The app name which starts the log.
   /// \param severity_threshold Logging threshold for the program.
-  /// \param logDir Logging output file name. If empty, the log won't output to file.
+  /// \param log_dir Logging output directory name.
+  /// \param log_file Logging output file name.
+  ///
+  /// Both [log_dir] and [log_file] are used to determine the output logging file.
```

Comment:
use backticks, not brackets

Reply:
Wondering do we have any code style recommendation?

Reply:
I updated the comment, but still want to know about the coding standing / documentation style.

Reply:
no specific style rules, but I often use backticks and never saw brackets in Ray.

Reply:
If it's coding consistency I'm fine with it, but backtick is usually used in bash script for execution (i.e. make -j`nproc`). To reduce mixed usage, I usually use `[]` for function arguments and referenced function names.